apps/testingaccessibility/src/pages/learn/index.tsx

```diff
@@ -223,7 +226,8 @@ const Learn: React.FC<{purchases: Purchase[]; product: SanityDocument}> = ({
                           )
                         })}
 
-                        {sections?.length > 0 && (
+                        {process.env.NEXT_PUBLIC_CERTIFICATE_ENABLED &&
```

Comment:
by default this env doesn't exist so they are off until it is defined (value doesn't matter)